### PR TITLE
ADR-1434 Update adjustment content

### PR DESCRIPTION
**Pull request by davidfes**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -875,33 +875,33 @@ spiritsQuestions.checkYourAnswersLabel.card4 = Other ingredients you’ve used
...
-declareAdjustmentQuestion.para1.list2 = over-declared alcoholic products on an earlier return and may have paid too much duty
-declareAdjustmentQuestion.p2 = You may be required to provide an explanation if declaring previously under-declared or over-declared alcoholic products.
-declareAdjustmentQuestion.p3 = You can also tell us about any alcoholic products you previously paid duty on and:
-declareAdjustmentQuestion.para1.list3 = have become spoilt and not fit to be consumed
-declareAdjustmentQuestion.para1.list4 = duty was originally paid at a reduced rate (draught relief) however they are no longer eligible for that rate after being repackaged for non-draught sale
-declareAdjustmentQuestion.para1.list5 = have now exported, warehoused for export or completed its planned destruction having previously notifying HMRC
-declareAdjustmentQuestion.error.required = Select yes if you need to tell us about any adjustments from previous returns
-declareAdjustmentQuestion.change.hidden = whether you need to make any adjustments from a previously submitted return
```
Comment (davidfes) [inline conf/messages.en:889]: deleted this but checking with Chiran who has an idea on where it might be used

Comment (davidfes) [reply conf/messages.en:889]: We can delete it. It'll be looked at in 1510 - and a new message created as needed.

File: conf/application.conf [hunk 2]
```diff
@@ -101,6 +101,7 @@ urls {
   businessTaxAccount   = "http://localhost:9020/business-account/"
   requestAccess        = "http://localhost:9555/enrolment-management-frontend/HMRC-AD-ORG/request-access-tax-scheme?continue=/business-account"
   feedbackFrontendBase = "http://localhost:9514"
+  exciseEnquiries      = "https://www.gov.uk/government/organisations/hm-revenue-customs/contact/excise-enquiries"
```
Comment (simondrugan16) [inline conf/application.conf:104]: all other urls here are for various different services that need to be changed in different envs. this url is static, therefore I think it should sit elsewhere, perhaps in Constants?

File: app/views/adjustment/WhenDidYouPayDutyView.scala.html [hunk 3]
```diff
@@ -25,10 +27,11 @@
     govukButton: GovukButton,
     paragraph: Paragraph,
     sectionHeading: SectionHeading,
-    pageHeading: PageHeading
+    pageHeading: PageHeading,
+    link: Link
 )
 
-@(form: Form[_], mode: Mode, adjustmentType: AdjustmentType)(implicit request: Request[_], messages: Messages)
+@(form: Form[_], mode: Mode, adjustmentType: AdjustmentType, exciseEnquiriesUrl: String)(implicit request: Request[_], messages: Messages)
```
Comment (simondrugan16) [inline app/views/adjustment/WhenDidYouPayDutyView.scala.html:34]: If url is in Constants (suggested in prev comment) then would not need to pass it in here

**Review by simondrugan16 — DISMISSED** [review]

**Review by simondrugan16 — APPROVED** [review]

**Review by sini-george — APPROVED** [review]

**Pull request merged**